From b13f1a6676402ed088de2d9b5911af43ef47dcc3 Mon Sep 17 00:00:00 2001
From: WeiguangTWK <weiguangtwk@outlook.com>
Date: Sat, 4 Jul 2026 14:26:24 +0800
Subject: [PATCH] Neko LineageOS: [i10n] Change NTP address to aliyun

Default address is not that reachable in China, for i10n purpose we changed it to more accessible Aliyun NTP provider

TODO: Migrate to vendor
Change-Id: If5b6005ffd4f2967cfb8b18a5e890ad91cee1e3d
---
 core/res/res/values/config.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/res/res/values/config.xml b/core/res/res/values/config.xml
index 755f32528..345529549 100644
--- a/core/res/res/values/config.xml
+++ b/core/res/res/values/config.xml
@@ -2780,7 +2780,7 @@
          Items must be in the form: "ntp://<host>[:port]"
          This is not a registered IANA URI scheme. -->
     <string-array translatable="false" name="config_ntpServers">
-        <item>ntp://time.android.com</item>
+        <item>ntp://ntp.aliyun.com</item>
     </string-array>
     <!-- SNTP client config: Timeout to wait for an NTP server response in milliseconds. -->
     <integer name="config_ntpTimeout">5000</integer>
-- 
2.43.0

